TempoWorld/Source/TempoWorld/Public/ActorControl.proto

```diff
@@ -0,0 +1,225 @@
+// Copyright Tempo Simulation, LLC. All Rights Reserved
+
+syntax = "proto3";
+
+import "TempoScripting/Empty.proto";
+import "TempoScripting/Geometry.proto";
+
+message SpawnActorRequest {
+  string type = 1;
+  bool deferred = 2;
+  TempoScripting.Transform transform = 3;
+  string relative_to_actor = 4;
+}
+
+message SpawnActorResponse {
+  string spawned_name = 1;
```

Comment:
Add spawned transform

Reply:
Done, and also added to the response of finish spawning (the transform can also be adjusted on finish spawning).